src/raygun.rum.js

```diff
@@ -346,6 +356,10 @@ var raygunRumFactory = function (window, $, Raygun) {
             }
         }
 
+        function createRequestId() {
+            self.requestId = randomKey(16);
```

Comment:
Does the request id need to be unique in sessions or globally (across all sessions)?

Reply:
The request ID wouldn't need to be unique to the session. So just globally